09 Environment/ProcedureForIssues.md

```diff
@@ -0,0 +1,66 @@
+
+# Procedure for creating issues
+
+
+*Here you will find a guide on how to create issues for projects (iterations), and how you can connect them to pull requests. There is a short text and a step by step guide which is longer.*
+
+
+### Kort
+
+
```

Comment:
Please add a paragraph about naming convention, if we do the same as HOEK its literally the name of the issue, just camel cased